packages/browser-wallet/src/background/recovery.ts

```diff
@@ -1,23 +1,71 @@
 import {
+    AccountInfo,
+    createCredentialV1,
     createIdentityRecoveryRequest,
+    CredentialInputV1,
+    CredentialRegistrationId,
     CryptographicParameters,
+    getAccountAddress,
+    HttpProvider,
     IdentityObjectV1,
     IdentityRecoveryRequestInput,
+    JsonRpcClient,
     Network,
     Versioned,
 } from '@concordium/web-sdk';
 import { ExtensionMessageHandler } from '@concordium/browser-wallet-message-hub';
 import { IdentityProviderIdentityStatus, IdentityTokenContainer, sleep } from 'wallet-common-helpers';
-import { Identity, CreationStatus, IdentityProvider } from '@shared/storage/types';
 import { BackgroundResponseStatus } from '@shared/utils/types';
-import { addIdentity } from './update';
+import { Identity, CreationStatus, IdentityProvider, WalletCredential } from '@shared/storage/types';
+import { storedCurrentNetwork } from '@shared/storage/access';
+import { addCredential, addIdentity } from './update';
 
 // TODO: increase max empty
 // How many empty identityIndices are allowed before stopping
 const maxEmpty = 3;
```

Comment:
Update this to be 10, and remove the TODO.

Reply:
Because 10 is a good sample size?

Reply:
Because I asked Lars to also use 10.